tests/playwright-test/store.spec.ts

```diff
@@ -127,44 +132,35 @@ test('should isolate store state between projects', async ({ runInlineTest }) =>
         projects: [
           {
             name: 'p1',
-            setupMatch: /.*store.setup.ts/
           },
           {
             name: 'p2',
-            setupMatch: /.*store.setup.ts/
           }
         ]
       };
     `,
-    'store.setup.ts': `
-      const { test, expect, store } = pwt;
-      test.projectSetup('should initialize store', async ({ }) => {
-        expect(await store.get('number')).toBe(undefined);
-        await store.set('number', 2022)
-        expect(await store.get('number')).toBe(2022);
-
-        expect(await store.get('name')).toBe(undefined);
-        await store.set('name', 'str-' + test.info().project.name)
-        expect(await store.get('name')).toBe('str-' + test.info().project.name);
-      });
-    `,
-    'a.test.ts': `
+    'a.spec.ts': `
       const { test, store } = pwt;
-      test('should get data from setup', async ({ }) => {
-        expect(await store.get('number')).toBe(2022);
-        expect(await store.get('name')).toBe('str-' + test.info().project.name);
+      const storageKey = name => test.info().project.name + '/' + name;
```

Comment:
I'd just drop this test, it doesn't really add value today.

Reply:
yeah, I kept it to demonstrate how inconvenient it is at the moment.